components/desktop/e/efl/Makefile

```diff
@@ -50,16 +51,17 @@ LDFLAGS  += $(JPEG_LDFLAGS)
 
 CONFIGURE_OPTIONS+= CFLAGS="$(CFLAGS) -D__EXTENSIONS__"
 CONFIGURE_OPTIONS+= --with-opengl=full
+CONFIGURE_OPTIONS+= --with-profile=release
 CONFIGURE_OPTIONS+= --enable-debug
 CONFIGURE_OPTIONS+= --enable-fribidi
 CONFIGURE_OPTIONS+= --enable-harfbuzz
+CONFIGURE_OPTIONS+= --disable-libeeze
 CONFIGURE_OPTIONS+= --enable-lua-old
 CONFIGURE_OPTIONS+= --enable-pulseaudio
 CONFIGURE_OPTIONS+= --disable-cserve
 CONFIGURE_OPTIONS+= --enable-physics
-CONFIGURE_OPTIONS+= --disable-gstreamer
```

Comment:
I usually prefer to explicit important options even if they have defaults because then the Makefile itself is informative and not bound to changes in default behaviour.
But since these options are likely not changing anymore I am fine with this.